### DISCUSSION TITLE
Playtime

### DISCUSSION THREAD

**Discussion by sgamesdev** [Leo-Corporation/Gavilya#452] [Q&A]
How do you track a game playtime if let's say it's a UE game, you have the basic exe Panicore.exe and that exe launches another one Panicore-Win64-Shipping.exe

**Comment by lpeyr** [comment]
At present, tracking the playtime is unfeasible when the initial executable initiates a separate executable under a different process name. To monitor playtime, users are required to manually input the newly launched process.
